SimplePolicy/Settings.cs

```diff
@@ -24,6 +27,9 @@ public Settings(IConfigurationSection section)
         {
             this.MaxTransactionsPerBlock = GetValueOrDefault(section.GetSection("MaxTransactionsPerBlock"), 500, p => int.Parse(p));
             this.MaxFreeTransactionsPerBlock = GetValueOrDefault(section.GetSection("MaxFreeTransactionsPerBlock"), 20, p => int.Parse(p));
+            this.MaxTransactionSize = GetValueOrDefault(section.GetSection("MaxTransactionSize"), 100000, p => int.Parse(p));
+            this.MaxFreeTransactionSize = GetValueOrDefault(section.GetSection("MaxFreeTransactionSize"), 400, p => int.Parse(p));
+            this.FeePerExtraByte = GetValueOrDefault(section.GetSection("FeePerExtraByte"), 0.00001M, p => decimal.Parse(p));
```

Comment:
I think there are two concepts here: (1) pay for priority (2) pay for blockchain use. This plugin tries to deal with the second case and avoid huge transactions being passed to the blockchain, what may be done intentionally using useless attributes. Price may be high, I don't know yet, but one thing we must think is that when a Deploy is performed, a huge transaction is passed, so price won't be 90 GAS anymore, but slightly (or much more). I guess contract limit is 65k right now, we need to calculate the cost impact in this common situation, or perhaps use a similar strategy to 10 free gas... since one is paying 90 GAS, it shouldn't pay more.
I guess some price is really necessary, proportional to tx usage because it compromises blockchain database proportionally, which has a different purpose to general tx network price, which is also different to system fees :)

Reply:
Currently most of the smart contracts deployed in MainNet are around **2000-3500 bytes** which adds an extra fee of **0.02 GAS**.
Now the bigger smart contract in MainNet is **24600 bytes** which would be **0.23 GAS**.

I believe that these amounts are perfectly acceptable for the projects that want to deploy a SC.